README.md

```diff
@@ -1,9 +0,0 @@
-<img align="right" width="250px" src="https://cloud.githubusercontent.com/assets/8995723/12703902/58324a9e-c846-11e5-9f8f-49326881efaf.gif"/>
```

Comment:
Even though we aren't actively using the _main_ the readme, can we please aim to keep it in?
It shouldn't cause a conflict with gh-pages and a repo without a readme is a little confusing for most people - always good to let people know what's going on and how they can contribute!

Reply:
Thanks, @iteles!

I did include an updated [readme.md](https://github.com/notshi/leeds/blob/f96af5d7257342eba5ce1bbc829fd26fb0b01ceb/readme.md) with this pr albeit lowercase renaming.

Sorry, I may have misunderstood what you meant!

Reply:
No, you're absolutely right, I must have missed it! 🤦🏻‍♀️

Looks great!

Reply:
Yay! *phew* Thank you :}